package org.example;

import java.io.IOException;

public class Test10 {
    public static void main(String[] args) {
//        try {
//            throw new IOException();
//        } catch (Exception e) {
//            System.out.println("In the Exception catch block");
//        } catch (IOException e) {
//            System.out.println("In the IOException catch block");
//        }
    }
}
